AppImageBuilder.yml

```diff
@@ -0,0 +1,50 @@
+version: 1
+script:
+  # Remove any previous build
+  - rm -rf AppDir  | true
+  # Make usr and icons dirs
+  - mkdir -p AppDir/usr/src/module
+  - mkdir -p AppDir/usr/share/icons/hicolor/32x32/apps/
+  # Copy the python application code into the AppDir
+  - cp -r src  AppDir/usr/src/module
+  - cp -r run.sh  AppDir/usr/src/module
+  - cp ./icon.png AppDir/usr/share/icons/hicolor/32x32/apps/
```

Comment:
icons are required for appimages. Ideally if we provide our own docker container we can bundle a `viam` icon that this can pull in (rather than shipping some random tiny png I found on the internet)